Mathlib/Algebra/Module/Zlattice.lean

```diff
@@ -85,6 +85,23 @@ lemma fundamentalDomain_pi_basisFun [Fintype ι] :
     fundamentalDomain (Pi.basisFun ℝ ι) = Set.pi Set.univ fun _ : ι ↦ Set.Ico (0 : ℝ) 1 := by
   ext; simp
 
+  /-- The Voronoi domain of the ℤ-lattice spanned by `b`. -/
+def VoronoiDomain : Set E := {m | ∀ v ∈ span ℤ (Set.range b), ‖m‖ ≤ ‖m - v‖}
+
+@[simp]
+theorem mem_VoronoiDomain {m : E} :
+    m ∈ VoronoiDomain b ↔ ∀ v ∈ span ℤ (Set.range b), ‖m‖ ≤ ‖m - v‖ := Iff.rfl
+
+theorem map_VoronoiDomain {F : Type*} [NormedAddCommGroup F] [NormedSpace K F] (f : E ≃ₗ[K] F) :
+    f '' (VoronoiDomain b) = VoronoiDomain (b.map f) := by
```

Comment:
Certainly you need [LinearIsometryEquiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/LinearIsometry.html#LinearIsometryEquiv) (`≃ₗᵢ[K]`)?

Reply:
Most likely but this is new territory for me in Mathlib so I am still trying to familiarize myself with the API and what would be needed.  

```
theorem map_VoronoiDomain {F : Type*} [NormedAddCommGroup F] [NormedSpace K F] (e : E ≃ₗᵢ[K] F) :
        e '' (VoronoiDomain b) = VoronoiDomain (b.map e.toLinearEquiv) := by 
    sorry
```

type checks but not sure this is really what I want to be proving...(I was mostly using the theorems above for `fundamentalDomain` as a guide but that obviously only goes so far).